File: Python/0265. 内务评分.py
# If you need to import additional packages or classes, please import here.

def func():
    # please define the python3 input here. 
    # For example: a,b = map(int, input().strip().split())
    # please finish the function body here.
    # please define the python3 output here. For example: print().
    while True:
        try:
            T = int(input())
            for _ in range(T):
                n = int(input())
                arr = list(map(int, input().split()))
                arr.sort()
                print(int(0.5 + sum(arr[1:-1])/(n - 2)))
            
        except EOFError:
            break
    
if __name__ == "__main__":
    func()

File: C++/0265. 内务评分.cpp
// we have defined the necessary header files here for this problem.
// If additional header files are needed in your program, please import here.

int main()
{  
  // please define the C++ input here. For example: int a,b; cin>>a>>b;;  
  // please finish the function body here.  
  // please define the C++ output here. For example:cout<<____<<endl; 
    int T;
    cin>>T;
    for(int t=0;t<T;++t) {
        int n;
        cin>>n;
        long temp;
        long sum = 0;
        long h = 0;
        long l = (1<<31) - 1;
        for(int i=0;i<n;++i) {
            cin>>temp;
            sum += temp;
            h = max(h, temp);
            l = min(l, temp);
        }
        cout<<int(0.5 + 1.0L * (sum - h - l)/ (n - 2))<<endl;
    }
    return 0;
}
